client/disputes/evidence/dispute-evidence-form.js

```diff
@@ -0,0 +1,192 @@
+/**
```

Comment:
This code used to be in the `client/disputes/evidence/index.js` file, but I moved it here to make that file a bit cleaner. I made some minor tweaks to the code to use data from the wp.data state tree, but it's mostly the same.

Reply:
👍 I think this can go further, and actually retrieve the selections (and possibly actions) via hooks here. Could probably be a later PR though – it will make more of a difference if/when we want to fetch the evidence separately (i.e. after 273-gh-Automattic/woocommerce-payments-server).

Reply:
That's a good point. I think I'll make that change in this PR. Seems like a good use of our time since we're doing this anyway 🤔

Reply:
I think this has been fixed as of c524da1bc897cb85dbc20dc2ac0f4e5f44e8dafc. Let me know what you think :)